mne/source_estimate.py

```diff
@@ -2289,7 +2292,7 @@ def compute_morph_matrix(subject_from, subject_to, vertices_from, vertices_to,
             continue
         m = sparse.eye(len(idx_use), len(idx_use), format='csr')
         morpher[hemi] = _morph_buffer(m, idx_use, e, smooth, n_vertices,
-                                      vertices_to[hemi], maps[hemi])
+                                      vertices_to[hemi], maps[hemi], warn=warn)
```

Comment:
This is an unrelated but very simple change to the morphing code to complete our use of the `warn` parameter. I can split into a separate PR if necessary, but this PR is only +65/-16 so hopefully it's not too bad.